frontend/src/app/components/acceleration/accelerations-list/accelerations-list.component.ts

```diff
@@ -50,17 +50,38 @@ export class AccelerationsListComponent implements OnInit, OnDestroy {
     if (!this.widget) {
       this.websocketService.want(['blocks']);
       this.seoService.setTitle($localize`:@@02573b6980a2d611b4361a2595a4447e390058cd:Accelerations`);
+
+      this.paramSubscription = this.route.params.pipe(
+        tap(params => {
+          this.page = +params['page'] || 1;
+          this.pageSubject.next(this.page);
+        })
+      ).subscribe();
+
+
+      this.keyNavigationSubscription = this.stateService.keyNavigation$.pipe(
+        tap((event) => {
```

Comment:
The subscription and `pageChange()` is triggered on every key press,

Solution:

`filter(() => event.key === nextKey || event.key === prevKey)`